packages/medical-domain/src/domains/repositories/statistics/insulin-statistics.service.ts

```diff
@@ -104,14 +104,20 @@ function getAutomatedAndManualBasalDuration(basalsData: Basal[], dateFilter: Dat
   const manualBasalDuration = manualBasal.reduce((accumulator, manualBasal) => accumulator + manualBasal.duration, 0)
   const automatedBasals = basalData.filter(automatedBasal => automatedBasal.subType === 'automated')
   const automatedBasalDuration = automatedBasals.reduce((accumulator, automaticBasal) => accumulator + automaticBasal.duration, 0)
-  const total = manualBasalDuration + automatedBasalDuration
+  const numOfDay = (dateFilter.end - dateFilter.start) / MS_IN_DAY
+  const automatedBasalPerDay = automatedBasalDuration / numOfDay
+  const manualBasalPerDay = manualBasalDuration / numOfDay
+  const total = automatedBasalPerDay + manualBasalPerDay
+  const automatedPercentage = Math.round(100 * automatedBasalPerDay / total)
+  const manualPercentage = Math.round(100 * manualBasalPerDay / total)
+
 
   return {
-    automatedBasalDuration,
-    manualBasalDuration,
-    automatedAndManualTotalDuration: Math.round(total),
-    automatedBasalInDays: (automatedBasalDuration / total) * TimeService.MS_IN_DAY,
-    manualBasalInDays: (manualBasalDuration / total) * TimeService.MS_IN_DAY
+    automatedBasal: automatedBasalPerDay ,
```

Comment:
I don't remember if I said to rename this variable to `automatedBasal` (if I did, mea culpa), but looking back, I thing the name `automatedBasalDuration` was better